### DISCUSSION TITLE
Allow "before" bar alignment in the candlesitck panel

### DISCUSSION THREAD

**Discussion by sylr** [grafana/grafana#68301] [Dashboards, Panels & Visualizations]
The OLCH bars are currently immune to the Graph Style / Bar Alignment setting.

It would be nice if that could be fixed.

Cheers!

cc @leeoniya.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82674. Feel free to continue the discussion around this there. Thank you!
